pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -575,7 +575,13 @@ func (m *kubeGenericRuntimeManager) computePodActions(pod *v1.Pod, podStatus *ku
 		// need to restart it.
 		if containerStatus == nil || containerStatus.State != kubecontainer.ContainerStateRunning {
 			if kubecontainer.ShouldContainerBeRestarted(&container, pod, podStatus) {
-				message := fmt.Sprintf("Container %+v is dead, but RestartPolicy says that we should restart it.", container)
+				var message string
+				if containerStatus == nil {
+					message = fmt.Sprintf("Starting container: %v", container)
```

Comment:
%+v will give field name

Reply:
Ups, thanks for noticing! Pushing again :)

Reply:
Fixed, thanks!